include/superior_mysqlpp/converters/to_integer.hpp

```diff
@@ -10,20 +10,43 @@
 #include <limits>
 #include <stdexcept>
 
-
+/*
+ * Namespace of converters designed for better performance
+ */
 namespace SuperiorMySqlpp { namespace Converters
 {
     namespace detail
     {
+        /**
+         * @brief Calculates the power of number.
+         * @tparam T Type of output and base.
+         * @param x Base.
+         * @param n Nonnegative exponent.
+         * @return Power of x.
+         */
         template <class T>
         inline constexpr T pow(T const& x, std::size_t n) {
             return n > 0 ? x * pow(x, n - 1) : 1;
         }
 
+        /**
+         * @brief Parses the string into the number of given type.
+         * @tparam T Type of result.
+         * @tparam length Length of the string.
+         * @tparam validate Whether to check if string is convertible.
+         */
         template<typename T, int length, bool validate>
         struct ToIntegerParserImpl
         {
+            /**
+             * Check if length is not greater than is allowed for given type.
+             */
             static_assert(std::numeric_limits<T>::digits10, "length > digits10");
+            /**
```

Comment:
It would be beneficial for better readability to have an empty line between `static_assert` and comment of `call` method

Reply:
ACK